subprojects/docs/src/docs/userguide/extending-gradle/task_configuration_avoidance.adoc

```diff
@@ -67,8 +67,8 @@ We also cover some <<#sec:task_configuration_avoidance_troubleshooting, troubles
 === General
 1. [[task_configuration_avoidance_guideline_use_help_task]] **Use `help` task as a benchmark during the migration.**
 The `help` task is the perfect candidate to benchmark your migration process.
-In a build that uses only the configuration avoidance API, https://scans.gradle.com/s/o7qmlmmrsfxz4/performance/configuration?openScriptsAndPlugins=WzFd[a build scan would show no tasks created immediately or created during configuration], and only the tasks that were actually executed would be created.
-Be mindful of <<#task_configuration_avoidance_pitfall_build_scan_plugin, the version of the build scan plugin in use>>.
+In a build that uses only the configuration avoidance API, https://scans.gradle.com/s/o7qmlmmrsfxz4/performance/configuration?openScriptsAndPlugins=WzFd[a Build Scan™ would show no tasks created immediately or created during configuration], and only the tasks that were actually executed would be created.
+Be mindful of <<#task_configuration_avoidance_pitfall_build_scan_plugin, the version of the Build Scan™ plugin in use>>.
```

Comment:
This is old advice. I don't think you can use a version of the plugin that this warning applies to and a new version of Gradle.
```suggestion
```